.github/workflows/main.yml

```diff
@@ -12,10 +12,17 @@ jobs:
         with:
           node-version: 12.x
       - name: Install Oasis
-        run: curl --proto '=https' --tlsv1.2 -sSL https://get.oasis.dev | python - --speedrun
+        run: curl --proto '=https' --tlsv1.2 -sSL https://get.oasis.dev | python - --speedrun --toolchain unstable
+      - name: Lint
+        run: |
+          pushd app
+          npm install
+          npm run-script lint
+          popd
       - name: Build and test service
         run: |
           export PATH=~/.local/bin:-~/.cargo/bin:$PATH
```

Comment:
you can add this to the circleci [environment](https://circleci.com/docs/2.0/configuration-reference/#environment) map

Reply:
I don't understand how this part is relevant to environment variables?

Reply:
exports are environment variables

Reply:
Oh silly me I was looking at the highlighted green text for running lint. Perhaps this should be changed in the cli or whatever unboxes `oasis init` since these lines are from that.